data/nation/alaska.ruleset

```diff
@@ -0,0 +1,112 @@
+[nation_alaska]
+
+name=_("Alaska")
+plural=_("?plural:Alaskan")
+groups="American"
+legend=_("Alaska is a state in the United States of America that was\
+ purchased from the Russian Empire in 1867, it became an organized\
+ territory on May 11th, 1912 and eventually gained statehood as the\
+ 49th state on January 3rd, 1959.")
+
+leaders = {
+ "name",                  "sex"
+ "Jefferson C. Davis",    "Male"   ; First chief federal officers of the Department of Alaska
+ "Henry E. Nichols",      "Male"   ; 20th and last chief federal officers of the Department of Alaska
+ "John Henry Kinkead",    "Male"   ; First Governor of the District of Alaska
+ "Walter Eli Clark",      "Male"   ; Last Governor of the District of Alaska and first Governor of the Territory of Alaska
+ "Mike Stepovich",        "Male"   ; Last Governor of the Territory of Alaska
+ "William A. Egan",       "Male"   ; First governor of the State of Alaska
+ "Wally Hickel",          "Male"   ; 2nd governor of the State of Alaska, only governor to be a member of the Alaskan Independece party
+ "Mike Gravel",           "Male"   ; Senator from Alaska and presidental canidate
+ "Sarah Palin",           "Female" ; 9th governor of the State of Alaska and vice presidental canidate
+ "Mike Dunleavy",         "Male"   ; 12th and current governor of the state of Alaska
+}
+
+ruler_titles = {
+ "government",      "male_title",      "female_title"
+ "Despotism",       _("Sheriff %s"),   _("?female:Sheriff %s")
+ "Republic",        _("Governor %s"),  _("?female:Governor %s")
+ "Fundamentalism",  _("Reverend %s"),  _("?female:Reverend %s")
+}
+
+flag="alaskan"
+flag_alt = "usa"
+style = "European"
+
+init_techs=""
+init_buildings=""
+init_units=""
+
+civilwar_nations="aleut", "american", "canadian"
+
+cities =
+ "Juneau",
+ "Anchorage",
+ "Fairbanks",
+ "Wasilia",
+ "Sitka",
+ "Ketchikan",
+ "Kenai",
+ "Bethel",
+ "Palmer",
+ "Kodiak",
+ "Homer",
+ "Utqiaġvik",     ;Barrow
+ "Soldotna",
+ "Unalaska",
+ "Valdez",
+ "Nome",
+ "Kotzebue",
+ "Seward",
+ "Cordova",
+ "Dillingham",
+ "North Pole",
```

Comment:
Really?

Reply:
Not sure what you are getting at,  if you are talking about Utqiaġvik then the Barrow comment is in reference to the fact that an alternate name for it is Barrow, or if you are talking about North Pole, that is an actual place in Alaska. https://en.wikipedia.org/wiki/North_Pole,_Alaska

Reply:
Oh, I didn't know it existed!